File: adjacentElementsProduct.py
def adjacentElementsProduct(inputArray):
    largest_product = inputArray[0]*inputArray[1]
    for index, item in enumerate(inputArray):
            print(index, item)
            if index+1 < len(inputArray):
                if item* inputArray[index+1] > largest_product:
                    largest_product = item * inputArray[index+1]
    return largest_product

File: adjacentElementsProduct.java
int adjacentElementsProduct(int[] inputArray) {
    int largestProduct = inputArray[0] * inputArray[1];
        for(int i = 0; i < inputArray.length-1; i++ ){
            int roundProduct = inputArray[i] * inputArray[i+1];
                if( roundProduct > largestProduct){
                    largestProduct = roundProduct;
                }
        }
    return largestProduct;
}